### PR TITLE
Fix MS13761: Change the move conditions under which overlays hide

### PR DESCRIPTION
**Pull request by zfox23**
Fixes [MS13761](https://highfidelity.fogbugz.com/f/cases/13761/Holding-CTRL-to-go-down-causes-overlays-to-disappear-even-when-I-m-on-the-ground-and-not-moving) by changing the conditions under which overlays hide from "is the user using a 'drive avatar' key" to "is the user moving".

### CODE REVIEW

**Comment by zfox23** [comment]
## Test Plan
1. Be in desktop mode. Place your avatar on a ground surface somewhere. Hold "C" (the "Fly Down" key).
    i. Verify that your HUD overlay does NOT disappear.
2. Walk around.
    i. Verify that, as you walk around, your HUD disappears and only returns once you stop moving.
3. Fly around.
    i. Verify that, as you fly around, your HUD disappears and only returns once you stop moving.

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10477/HighFidelity-Beta-PR13341-812c28e.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10477/HighFidelity-Beta-Interface-PR13341-812c28e.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10477/HighFidelity-Beta-PR13341-812c28e.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10477/HighFidelity-Beta-Interface-PR13341-812c28e.exe

**Comment by qligmaloney** [comment]
Currently testing

**Comment by qligmaloney** [comment]
PASS @conklin94122 Overlay disappears when walking/flying and reappears after avatar stops

File: interface/src/ui/OverlayConductor.cpp [hunk 1]
```diff
@@ -115,20 +88,21 @@ void OverlayConductor::update(float dt) {
         _hmdMode = false;
     }
 
-    bool prevDriving = _currentDriving;
-    bool isDriving = updateAvatarHasDriveInput();
-    bool drivingChanged = prevDriving != isDriving;
     bool isAtRest = updateAvatarIsAtRest();
+    bool prevMoving = _currentMoving;
```
Comment (danteruiz) [inline interface/src/ui/OverlayConductor.cpp]: Does `_currentMoving` ever get changed? If not what is it used for?

Comment (zfox23) [reply interface/src/ui/OverlayConductor.cpp]: Ah, thank you for calling this out. I was following the previous model without thinking. Lazy!

**Comment by conklin94122** [comment]
@zfox23 I'm removing QA Approved due to requested dev changes. Please relabel as QA Ready when it is, or let me know that retesting is entirely unnecessary.

**Comment by zfox23** [comment]
@conklin94122 - QA retesting is not necessary; the code changes were purely cleanup and not logical changes.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10508/HighFidelity-Beta-PR13341-c26b03f.apk

File: interface/src/ui/OverlayConductor.cpp [hunk 2]
```diff
@@ -69,31 +67,6 @@ bool OverlayConductor::updateAvatarIsAtRest() {
     return _currentAtRest;
 }
 
-bool OverlayConductor::updateAvatarHasDriveInput() {
```
Comment (danteruiz) [inline interface/src/ui/OverlayConductor.cpp]: You did not remove the definition of this function from the header file. (I am surprised that this even compiled)

Comment (zfox23) [reply interface/src/ui/OverlayConductor.cpp]: thanks...

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/10510/HighFidelity-Beta-PR13341-56bd7e3.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10510/HighFidelity-Beta-PR13341-56bd7e3.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/10510/HighFidelity-Beta-Interface-PR13341-56bd7e3.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10510/HighFidelity-Beta-PR13341-56bd7e3.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/10510/HighFidelity-Beta-Interface-PR13341-56bd7e3.exe

**Pull request merged**